doc/logging/persistent_logger/persistent_loglevel.md

```diff
@@ -0,0 +1,217 @@
+# Persistent logger HLD
+
+# High Level Design Document
+
+#### Rev 0.1
+
+# Table of Contents
+- [Persistent logger HLD](#persistent-logger-hld)
+- [High Level Design Document](#high-level-design-document)
+      - [Rev 0.1](#rev-01)
+- [Table of Contents](#table-of-contents)
+- [List of Tables](#list-of-tables)
+- [List of Figures](#list-of-figures)
+- [Revision](#revision)
+- [Scope](#scope)
+- [Motivation](#motivation)
+- [Definitions/Abbreviation](#definitionsabbreviation)
+- [1 Background](#1-background)
+  - [1.1 "swssloglevel" usage](#11-"swssloglevel"-usage)
+  - [1.2 LOGLEVEL DB schema](#12-loglevel-db-schema)
+- [2 Requirements Overview](#2-requirements-overview)
+- [3 Persistent log level design](#3-persistent-log-level-design)
+  - [3.1 High-level design](#31-high-level-design)
+  - [3.2 Persistent logger flow](#32-persistent-logger-flow)
+    - [3.2.1 Return to default log level](#321-return-to-default-log-level)
+- [4 Flows](#4-flows)
+  - [4.1 Logger init flow](#41-logger-init-flow)
+  - [4.2 "config reload" and "config save" flows](#42-"config-reload"-and-"config-save"-flows)
+- [5 Warm Boot Support](#5-warm-boot-support)
+  - [5.1 Keep log level persistent to warm-boot automatic](#51-keep-log-level-persistent-to-warm-boot-automatic)
+  - [5.2 Make the log level persistent to warm-boot only by command](#52-make-the-log-level-persistent-to-warm-boot-only-by-command)
+- [6 Fast Boot Support](#6-fast-boot-support)
+- [7 Open Questions](#8-open-questions)
+  - [7.1 Log level persistency in warm-boot](#81-log-level-persistency-in-warm-boot)
+
+
+# List of Tables
+* [Table 1: Abbreviations](#definitionsabbreviation)
+
+
+# List of Figures
+* [persistent logger flow](#321-return-to-default-log-level)
+
+
+# Revision
+| Rev | Date     | Author          | Change Description                 |
+|:---:|:--------:|:---------------:|------------------------------------|
+| 0.1 | 06/21/22 | Eden Grisaro    | Initial version                    |
+
+# Scope
+This document provides an overview of the implementation of making the SWSS Logger persistent for SWSS, Syncd, and SAI components.
```

Comment:
This document provides high level design for SWSS Logger - log level persistent for SWSS, Syncd, and SAI components

Reply:
Done